charts/datadog/values.yaml

```diff
@@ -624,6 +624,11 @@ datadog:
     # datadog.processAgent.processDiscovery -- Enables or disables autodiscovery of integrations
     processDiscovery: true
 
+    # datadog.processAgent.runInCoreAgent -- Set this to true to run the following features in the core agent: Live Processes, Live Containers, Process Discovery.
+    ## This is an experimental feature requiring Agent 7.53.0+ and Linux. Currently not compatible with APM Single Step Instrumentation.
+    ## If Orchestrator Explorer, Network Performance Monitoring is not being used, processAgent.enabled should be set to false to remove the process-agent container.
```

Comment:
to clarify, if a user is using NPM and `runInCoreAgent: true`, then `processAgent.enabled` still should be `true` ?

Reply:
Yes